docs/machine-restore.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Machine restore
+---
+
+The Studio is built using a server-client architecture. Whenever you edit a machine, we will persist these changes to our database, ready for you to use when you return - on any device with an internet connection.
+
+![Simplified drawing of the Stately Studio architecture described above.](/studio_architecture_challenges.png)
+
+But what happens with your edits if your internet connection fails or the Stately backend is having problems? This is where machine restore comes into play.
+
+### Example
+
+In our example, Santa’s delivery machine has a problem; the Christmas-time state is unreachable. So we do as any good Santa’s helper would; we create an event to get to Christmas. But alas, we’re offline, so our changes could get lost. Before this new feature, there was a chance you wouldn’t notice you’d gone offline or that the server returned an error. But as you can see in the GIF below, we’ve introduced new error banners to show you what’s happening.
```

Comment:
Nice seasonal example 😆 

Could we add some text for what the banner says? Just in case someone can’t read it from the gif. Also it would help if someone gets the banner in the Studio and searches for the text in the docs to understand why it happened.

Reply:
Thanks - good idea, adding the text now.